<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Tomeu Borras Riera</author>
 <class>BfConfiguracionBase</class>
 <widget class="QWidget" name="BfConfiguracionBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Datos de la empresa</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../bulmages.qrc">
    <normaloff>:/Images/document-properties.png</normaloff>:/Images/document-properties.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer name="spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>411</width>
         <height>27</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="_3">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../bulmages.qrc">:/Images/document-properties.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Configuracion basica</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="margin">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Nombre:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_nombre</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="m_nombre"/>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>C.I.F.:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_cif</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLineEdit" name="m_cif"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Direccion:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_direccion</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="m_direccion"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>C.P.:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_cp</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="m_cp"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Municipio:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_municipio</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="m_municipio"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Provincia:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_provincia</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="5">
        <widget class="QLineEdit" name="m_provincia"/>
       </item>
       <item row="10" column="2" rowspan="6">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>278</width>
           <height>66</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="10" column="5" rowspan="6">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>208</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="5">
        <widget class="QLineEdit" name="m_ciudad"/>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Ciudad</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="m_pais"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Pais:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_pais</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QLineEdit" name="m_codArticuloGenerico"/>
       </item>
       <item row="6" column="3">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Codigo articulo generico:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>I.R.P.F.:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Telefono:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_telefono</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="m_telefono"/>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="m_fax"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fax:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_fax</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="5">
        <widget class="QLineEdit" name="m_serieFacturaDefecto"/>
       </item>
       <item row="7" column="3">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Serie factura defecto:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>E-Mail:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_mail</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="m_mail"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Web:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buddy">
          <cstring>m_web</cstring>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLineEdit" name="m_web"/>
       </item>
       <item row="8" column="3">
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Almacen defecto:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="5">
        <widget class="QLineEdit" name="m_almacenDefecto"/>
       </item>
       <item row="4" column="5">
        <widget class="BlDoubleSpinBox" name="m_irpf">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Configuracion avanzada</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="BfConfiguracionSubForm" name="mui_listado"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="2" margin="6"/>
 <customwidgets>
  <customwidget>
   <class>BfConfiguracionSubForm</class>
   <extends>QTableWidget</extends>
   <header>bfconfiguracionview.h</header>
  </customwidget>
  <customwidget>
   <class>BlDoubleSpinBox</class>
   <extends>QSpinBox</extends>
   <header>bldoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
